packages/styling/src/styles/IconCodes.ts

```diff
@@ -2951,3 +2951,11 @@ export const IconCodes = {
   eventDeclined: '\uF425',
 
 };
+
+/**
+ * Get IconCode by IconName.
+*/
+export function getIconCodeByIconName(iconName: string): string {
```

Comment:
Cool! Could you please tell me where and how to use your new added icons package in your PR?